.travis.yml

```diff
@@ -8,15 +8,16 @@ env:
   - BROWSER=ChromeCi
 
 cache:
-  directories:
-    - node_modules
+  - npm
```

Comment:
Why npm?

Reply:
node_modules

Reply:
using `yarn` alone should be sufficient then?

Reply:
unfortunately both are required: https://docs.travis-ci.com/user/caching/#yarn-cache

Reply:
yarn actually just takes everything from its cache dir anyway (instead of re-downloading). it's the same w/how travis doesn't cache `node_modules` when using `npm ci`